src/server/home/__snapshots__/controller.test.js.snap

```diff
@@ -202,27 +202,18 @@ exports[`#homeController content checks when auth is enabled should match conten
           take longer if veterinary checks are needed, or if we need further
           information.
         </p>
-          <h3 class="govuk-heading-m">Logging into the service</h3>
-          <p class="govuk-body">
-            You need a Defra account to use this service. You can register for
-            one using your Government Gateway log in details.
-          </p>
+          <h3 class="govuk-heading-m">Get help</h3>
           <p class="govuk-body">
-            After you log in you need to enter your contact information, and
-            details about your business if you have one.
+            If you need help using the service you can call 03000 200 301 and
```

Comment:
if `authRequired` is true in the dev environment, aren't we expecting this snapshot to remain unchanged?